site/content/en/blog/releases/1.22.0-rc

```diff
@@ -0,0 +1,49 @@
+---
+title: "1.22.0 - Release Candidate"
+linkTitle: "1.22.0-rc"
+date: "2022-03-22"
+---
+This is the 1.22.0-rc release of Agones.
+
+This Release  fixes the issue with  agones-images on ARM systems, multi-cluster allocation example solution to leverage GCP resources to connect to multiple Agones clusters and also Cert manager integration
```

Comment:
nit: We have some double spaces on this line. Also I'm unsure if Release should be capitalised?